src/docs/product/reference/search/searchable-properties/user-feedback.mdx

```diff
@@ -0,0 +1,143 @@
+---
+title: User Feedback Properties
+sidebar_order: 40
+description: "Learn more about searchable user feedback properties."
+redirect_from:
+  - /product/sentry-basics/search/searchable-properties/user-feedback/
+---
+
+<Include name="searchable-properties/intro-for-user-feedback.mdx" />
+
+## Searchable Properties
+
+Below is a list of keys and tokens that can be used in the user feedback search.
+
+<DefinitionList>
+
+### `assigned`
+
+Returns user feedback submissions assigned to the defined user(s) or team(s). Values can be a user ID (your email address), `me` for yourself, `none` for no assignee, `my_teams` or `#team-name` for teams you belong to.
+
+- **Type:** team or org user
+
+### `browser.name`
+
+Name of the browser.
+
+- **Type:** string
+
+### `device.brand`
+
+Brand of the device.
+
+- **Type:** string
+
+### `device.family`
+
+Family of the device. Typically, the common part of a model name across generations. For example, iPhone, Samsung Galaxy.
+
+- **Type:** string
+
+### `device.model_id`
+
+Internal hardware revision to identify the device exactly.
+
+- **Type:** n/a
+
+### `device.name`
+
+Name of the device.
+
+- **Type:** string
+
+### `dist`
+
+Distinguishes build or deployment variants of the same release of an application. For example, the dist can be the build number of an XCode build or the version code of an Android build.
```

Comment:
tiny nit: `Xcode` instead of `XCode` :)

Reply:
ahh, this is in a couple places in the docs too. i'll fix those other places as well